File: NetFramework/Gauge/VB/GaugeDemo/PageYandexGauge.vb
Imports C1.Win.C1Gauge

Public Class PageYandexGauge

    Private Sub c1RadialGauge1_ItemMouseMove(ByVal sender As System.Object, ByVal e As ItemMouseEventArgs) Handles c1RadialGauge1.ItemMouseMove, c1RadialGauge1.ItemMouseDown
        If e.Button = System.Windows.Forms.MouseButtons.Left Then
            Dim p As C1GaugePointer = e.Gauge.Pointer
            p.Value = p.GetValueAt(e.X, e.Y)
        End If
    End Sub

End Class

File: Next/Gauge/CS/GaugeDemo/PageYandexGauge.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using C1.Win.Gauge;

namespace GaugeDemo
{
    public partial class PageYandexGauge : GaugeDemo.BaseGaugePage
    {
        public PageYandexGauge()
        {
            InitializeComponent();
        }

        private void c1RadialGauge1_ItemMouseMove(object sender, C1.Win.Gauge.ItemMouseEventArgs e)
        {
            if (e.Button == MouseButtons.Left)
            {
                C1GaugePointer p = e.Gauge.Pointer;
                p.Value = p.GetValueAt(e.X, e.Y);
            }
        }
    }
}
